Modules/Sources/NetworkingCore/Extensions/String+HTML.swift

```diff
@@ -0,0 +1,14 @@
+import HTMLParser
+
+/// String: HTML Stripping
+///
+extension String {
+
+    /// Returns the HTML Stripped version of the receiver.
+    ///
+    /// NOTE: I can be very slow ⏳ — using it in a background thread is strongly recommended.
+    ///
+    public var strippedHTML: String {
+        HTMLParser().parse(self).rawText()
+    }
+}
```

Comment:
Looks new, but it's just a big diff for Git to follow:

https://github.com/woocommerce/woocommerce-ios/blob/c8788efc3e4aa44e1786c981923218582b06cf53/Networking/Sources/Core/Extensions/String%2BHTML.swift#L1-L24